packages/contracts-bedrock/src/L1/OptimismPortal.sol

```diff
@@ -93,55 +91,71 @@ contract OptimismPortal is Initializable, ResourceMetering, ISemver {
     }
 
     /// @notice Semantic version.
-    /// @custom:semver 2.3.0
-    string public constant version = "2.3.0";
+    /// @custom:semver 2.4.0
+    string public constant version = "2.4.0";
 
     /// @notice Constructs the OptimismPortal contract.
-    /// @param _l2Oracle Address of the L2OutputOracle contract.
-    /// @param _systemConfig Address of the SystemConfig contract.
-    constructor(L2OutputOracle _l2Oracle, SystemConfig _systemConfig) {
-        L2_ORACLE = _l2Oracle;
-        SYSTEM_CONFIG = _systemConfig;
-        initialize(SuperchainConfig(address(0)));
+    constructor() {
+        initialize({
+            _l2Oracle: L2OutputOracle(address(0)),
+            _systemConfig: SystemConfig(address(0)),
+            _superchainConfig: SuperchainConfig(address(0))
+        });
     }
 
     /// @notice Initializer.
-    /// @param _superchainConfig Address of the SuperchainConfig contract.
-    function initialize(SuperchainConfig _superchainConfig) public initializer {
+    /// @param _l2Oracle Contract of the L2OutputOracle.
+    /// @param _systemConfig Contract of the SystemConfig.
+    /// @param _superchainConfig Contract of the SuperchainConfig.
+    function initialize(
+        L2OutputOracle _l2Oracle,
+        SystemConfig _systemConfig,
+        SuperchainConfig _superchainConfig
+    )
+        public
+        initializer
+    {
         l2Sender = Constants.DEFAULT_L2_SENDER;
```

Comment:
Do we want the same thing done here with `l2Sender` as in https://github.com/ethereum-optimism/optimism/pull/8797? cc @maurelian @mds1

Reply:
Yes, we should.
But I will likely address that as an audit fix. 

Would it be possible to hold off on merging this PR, until the extended pause audit fixes are in. I can push to make them all ASAP to unblock this.

Reply:
just to confirm: no changes here in this PR?

Reply:
No, the changes @tynes referenced were handled in #8797. 
Please make sure the feature branch is rebased on it, and that you're maintaining those changes.